### PR TITLE
Fix Chebyshev rate constant when using local quantity units

### PR DESCRIPTION
**Pull request by speth**
Since the leading Chebyshev coefficient has effective units like log(cm^3/kmol), it needs to be converted directly to the default units of the CTI file.

Analogous to the fix for PLOG reactions in #435.

### CODE REVIEW

**Comment by bryanwweber** [comment]
Hey @speth, just curious what you changed to fix the previous CI errors so I can learn from it :smile:

**Comment by speth** [comment]
The first version of this didn't update all of the affected tests. I hadn't noticed the failure because I was only running a subset of the test suite locally.

File: interfaces/cython/cantera/ck2cti.py [hunk 1]
```diff
@@ -65,6 +65,8 @@
                 'KJOULES/MOL': 'kJ/mol',
                 'KJOULES/MOLE': 'kJ/mol'}
 
+Avogadro = 6.02214129e23  # in molec/mol
```
Comment (bryanwweber) [inline interfaces/cython/cantera/ck2cti.py]: Can you document where this number comes from? For instance, it is different from the CODATA value: https://physics.nist.gov/cgi-bin/cuu/Value?na

**Review by bryanwweber — APPROVED** [review]

File: interfaces/cython/cantera/ck2cti.py (hunk 1 above)
Comment (speth) [reply interfaces/cython/cantera/ck2cti.py]: It is the 2010 CODATA value, consistent with the value in `ct_defs.h`.

**Pull request merged**